components/LessonCard.tsx

```diff
@@ -104,7 +104,7 @@ const LessonCard: React.FC<Props> = props => {
           </div>
           <div className="col-9">
             <h4 className="lesson-card__title font-weight-bold mt-3">
-              {props.title}
+              <a href={`/curriculum/${props.lessonId}`}>{props.title}</a>
```

Comment:
If this is internal. It should be a `<Link>`

Reply:
Fixed :0